runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -668,4 +676,23 @@ private void confirmKickPlayer(final String kickPlayerName)
 			.option("2. Cancel", Runnables::doNothing)
 			.build();
 	}
+
+	private void colorIgnoredPlayers()
+	{
+		if (client.getWidget(WidgetInfo.CLAN_CHAT_LIST).getChildren() == null)
+		{
+			return;
+		}
+
+		Color ignoreColor = config.showIgnores() ? config.showIgnoresColor() : Color.WHITE;
+		for (Widget listWidget : client.getWidget(WidgetInfo.CLAN_CHAT_LIST).getChildren())
+		{
+			if (listWidget.getText().isEmpty() || listWidget.getText().matches("World \\d+") ||
```

Comment:
Is there a better way to detect this? Some people have an rsn matching `World \\d+`

Reply:
Added so it hops every 3 widgets, since the order that the widgets go in is name, world, icon.